### PR TITLE
To update all the tests in numba_dppy/tests/dppl

### PR DESCRIPTION
**Pull request by vlad-perevezentsev**
I updated all the tests.
In tests dppl_fallback, parfor_lower_message, prange, with_context use functions from testing.py. Classes in testing.py were deleted.

### CODE REVIEW

File: numba_dppy/examples/pa_examples/test1-2d.py [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-a = np.ones((N,N), dtype=np.float32)
-b = np.ones((N,N), dtype=np.float32)
+a = np.ones((N, N), dtype=np.float32)
+b = np.ones((N, N), dtype=np.float32)
 
 print("a:", a, hex(a.ctypes.data))
 print("b:", b, hex(b.ctypes.data))
-c = f1(a,b)
+
+with dpctl.device_context():
```
Comment (diptorupd) [inline numba_dppy/examples/pa_examples/test1-2d.py:21]: Let us use explicit device selection *e.g.*, `with dpctl.device_context("opencl:gpu:0"):`

File: numba_dppy/examples/pa_examples/test1-3d.py [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
-a = np.ones((N,N,N), dtype=np.float32)
-b = np.ones((N,N,N), dtype=np.float32)
+a = np.ones((N, N, N), dtype=np.float32)
+b = np.ones((N, N, N), dtype=np.float32)
 
 print("a:", a, hex(a.ctypes.data))
 print("b:", b, hex(b.ctypes.data))
-c = f1(a,b)
+
+with dpctl.device_context():
```
Comment (diptorupd) [inline numba_dppy/examples/pa_examples/test1-3d.py:21]: Same change here.

File: numba_dppy/examples/pa_examples/test1-4d.py [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
-a = np.ones((N,N,N,N), dtype=np.float32)
-b = np.ones((N,N,N,N), dtype=np.float32)
+a = np.ones((N, N, N, N), dtype=np.float32)
+b = np.ones((N, N, N, N), dtype=np.float32)
 
 print("a:", a, hex(a.ctypes.data))
 print("b:", b, hex(b.ctypes.data))
-c = f1(a,b)
+
+with dpctl.device_context():
```
Comment (diptorupd) [inline numba_dppy/examples/pa_examples/test1-4d.py:21]: same here

File: numba_dppy/examples/pa_examples/test1-5d.py [hunk 4]
```diff
@@ -1,26 +1,32 @@
...
-a = np.ones((N,N,N,N,N), dtype=np.float32)
-b = np.ones((N,N,N,N,N), dtype=np.float32)
+a = np.ones((N, N, N, N, N), dtype=np.float32)
+b = np.ones((N, N, N, N, N), dtype=np.float32)
 
 print("a:", a, hex(a.ctypes.data))
 print("b:", b, hex(b.ctypes.data))
-c = f1(a,b)
+
+with dpctl.device_context():
```
Comment (diptorupd) [inline numba_dppy/examples/pa_examples/test1-5d.py:21]: same here.

File: numba_dppy/examples/pa_examples/test1.py [hunk 5]
```diff
@@ -19,7 +20,10 @@ def main():
 
     print("a:", a, hex(a.ctypes.data))
     print("b:", b, hex(b.ctypes.data))
-    c = f1(a,b)
+
+    with dpctl.device_context():
```
Comment (diptorupd) [inline numba_dppy/examples/pa_examples/test1.py:24]: same here.

File: numba_dppy/tests/test_dpnp_functions.py [hunk 6]
```diff
@@ -66,6 +78,7 @@ def test_for_dimensions(fn, test_fn, dims, tys, np_all=False):
 
     return True
 
+
 def ensure_dpnp():
     try:
        # import dpnp
```
Comment (diptorupd) [inline numba_dppy/tests/test_dpnp_functions.py:84]: Please remove commented out old code.

**Review by diptorupd — APPROVED** [review]
LGTM except minor nitpicks. @PokhodenkoSA please merge after @vlad-perevezentsev has fixed the minor things and verifying cause of Windows CI failure.

**Comment by vlad-perevezentsev** [comment]
@reazulhoque Do we need to import numba-dppy in tests? It is not explicitly used in 
test_dppl_fallback,
test_numpy_bit_twiddling_functions,
test_numpy_comparison_functions,
test_numpy_floating_functions,
test_numpy_math_functions, 
test_numpy_trigonomteric_functions, 
test_vectorize

Tests work without it.

**Pull request merged**